Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -359,10 +359,235 @@ C_130J:defineTumb("AUG_ICS_TRANSMISSION_SELECTOR", devices.VOLUME_MANAGER, 3144,
+C_130J:definePushButton("AUG_CNI_KBD_G", devices.AC_CNI, 3048, 1286, AUG_CNI_MU, "Aug Crew CNI-MU G Key")
+C_130J:definePushButton("AUG_CNI_KBD_H", devices.AC_CNI, 3049, 1287, AUG_CNI_MU, "Aug Crew CNI-MU H Key")
+C_130J:definePushButton("AUG_CNI_KBD_I", devices.AC_CNI, 3050, 1288, AUG_CNI_MU, "Aug Crew CNI-MU I Key")
+C_130J:definePushButton("AUG_CNI_KBD_J", devices.AC_CNI, 3051, 1289, AUG_CNI_MU, "Aug Crew CNI-MU J Key")
+C_130J:definePushButton("AUG_CNI_KBD_K", devices.AC_CNI, 3052, 1290, AUG_CNI_MU, "Aug Crew CNI-MU K Key")
+C_130J:definePushButton("AUG_CNI_KBD_L", devices.AC_CNI, 3053, 1291, AUG_CNI_MU, "Aug Crew CNI-MU L Key")
+C_130J:definePushButton("AUG_CNI_KBD_M", devices.AC_CNI, 3054, 1292, AUG_CNI_MU, "Aug Crew CNI-MU M Key")
+C_130J:definePushButton("AUG_CNI_KBD_N", devices.AC_CNI, 3055, 1293, AUG_CNI_MU, "Aug Crew CNI-MU N Key")
+C_130J:definePushButton("AUG_CNI_KBD_O", devices.AC_CNI, 3056, 1294, AUG_CNI_MU, "Aug Crew CNI-MU O Key")
+C_130J:definePushButton("AUG_CNI_KBD_P", devices.AC_CNI, 3057, 1295, AUG_CNI_MU, "Aug Crew CNI-MU P Key")
+C_130J:definePushButton("AUG_CNI_KBD_Q", devices.AC_CNI, 3058, 1296, AUG_CNI_MU, "Aug Crew CNI-MU Q Key")
+C_130J:definePushButton("AUG_CNI_KBD_R", devices.AC_CNI, 3059, 1297, AUG_CNI_MU, "Aug Crew CNI-MU R Key")
+C_130J:definePushButton("AUG_CNI_KBD_S", devices.AC_CNI, 3060, 1298, AUG_CNI_MU, "Aug Crew CNI-MU S Key")
+C_130J:definePushButton("AUG_CNI_KBD_T", devices.AC_CNI, 3061, 1299, AUG_CNI_MU, "Aug Crew CNI-MU T Key")
+C_130J:definePushButton("AUG_CNI_KBD_U", devices.AC_CNI, 3062, 1300, AUG_CNI_MU, "Aug Crew CNI-MU U Key")
+C_130J:definePushButton("AUG_CNI_KBD_V", devices.AC_CNI, 3063, 1301, AUG_CNI_MU, "Aug Crew CNI-MU V Key")
+C_130J:definePushButton("AUG_CNI_KBD_W", devices.AC_CNI, 3064, 1302, AUG_CNI_MU, "Aug Crew CNI-MU W Key")
+C_130J:definePushButton("AUG_CNI_KBD_X", devices.AC_CNI, 3065, 1303, AUG_CNI_MU, "Aug Crew CNI-MU X Key")
+C_130J:definePushButton("AUG_CNI_KBD_Y", devices.AC_CNI, 3066, 1304, AUG_CNI_MU, "Aug Crew CNI-MU Y Key")
+C_130J:definePushButton("AUG_CNI_KBD_Z", devices.AC_CNI, 3067, 1305, AUG_CNI_MU, "Aug Crew CNI-MU Z Key")
+C_130J:definePushButton("AUG_CNI_KBD_SLASH", devices.AC_CNI, 3068, 1308, AUG_CNI_MU, "Aug Crew CNI-MU / Key")
+C_130J:definePushButton("AUG_CNI_KBD_DEL", devices.AC_CNI, 3028, 1307, AUG_CNI_MU, "Aug Crew CNI-MU DEL Key")
+C_130J:definePushButton("AUG_CNI_KBD_CLR", devices.AC_CNI, 3029, 1309, AUG_CNI_MU, "Aug Crew CNI-MU CLR Key")
+C_130J:defineIndicatorLight("AUG_CNI_DSPY_LED", 4145, AUG_CNI_MU, "Aug Crew CNI-MU DSPY Light (Green)")
+C_130J:defineIndicatorLight("AUG_CNI_MSG_LED", 4146, AUG_CNI_MU, "Aug Crew CNI-MU MSG Light (Green)")
+C_130J:defineIndicatorLight("AUG_CNI_FAIL_LED", 4147, AUG_CNI_MU, "Aug Crew CNI-MU FAIL Light (Yellow)")
+C_130J:defineIndicatorLight("AUG_CNI_OFSET_LED", 4148, AUG_CNI_MU, "Aug Crew CNI-MU OFSET Light (Green)")
+C_130J:defineIndicatorLight("AUG_CNI_EXEC_LED", 3394, AUG_CNI_MU, "Aug Crew CNI-MU EXEC Light (Green)")
```

Comment:
`AUG_CNI_EXEC_LED` should be a defineFloat as the light is dimmable and it's brightness is linked to this draw arg.